### DISCUSSION TITLE
Gift Cards

### DISCUSSION THREAD

**Discussion by Deni-Iconis** [saleor/saleor#7554] [Q&A] [answered]
Is there any documentation regarding Gift Cards? Can't seem to find any.. Wandering if it's OK to use them or are they gonna get removed in feature releases?

**Comment by karolkielecki** [comment, marked as answer]
Hi! Our API supports gift cards, but the dashboard and storefront don't support this feature yet. We aim to deliver gift cards in tandem with Saleor 3.1 https://github.com/mirumee/saleor/projects/12

**Reply by Deni-Iconis** [reply]
I missed that part in 'projects', thank you!
